### PR TITLE
Feature/profile page architecture

### PR DESCRIPTION
**Pull request by miinow**
settings of Profile page, Radio&RadioGroup Components
___
NB - by making a pull request you confirm agreement to the [Contributors Agreement](https://github.com/openlawnz/openlawnz-web/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: src/jsx/components/Modal/Radio.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	return (
+		<div className="radio-wrapper">
+			<div className="radio-combo">
+				<input readOnly checked={!!checked} type="radio" id={value} value={value} name="age" />
+				<span className="radio-target" />
+			</div>
+			<label className="radio-label" htmlFor={value}>
+				{value}
+			</label>
+			{subValue ? <span className="radio-sub-value">{`(${subValue})`}</span> : null}
```
Comment (caionnx) [inline src/jsx/components/Modal/Radio.jsx]: Could we display this `subvalue` combined with label? For screen readers would be all read as one information.

File: src/jsx/components/Modal/ProfileEdit.jsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+					onCancel={this.onCancel}
+					clickMask={this.clickMask}
+					reset={this.reset}
+					visible={this.state.visible}
+					id={this.props.id}
+					key={this.props.id}
+				>
+					<div className="profile-modal-header">Edit Folder</div>
+					<div className="profile-modal-input-wrapper">
+						<span>Folder Name: </span>
```
Comment (caionnx) [inline src/jsx/components/Modal/ProfileEdit.jsx]: Instead of span, lets use label

File: src/jsx/components/Modal/ProfileEdit.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+					onCancel={this.onCancel}
+					clickMask={this.clickMask}
+					reset={this.reset}
+					visible={this.state.visible}
+					id={this.props.id}
+					key={this.props.id}
+				>
+					<div className="profile-modal-header">Edit Folder</div>
+					<div className="profile-modal-input-wrapper">
+						<label>Folder Name: </label>
```
Comment (caionnx) [inline src/jsx/components/Modal/ProfileEdit.jsx]: Hi Nicolas, missing the `for` attribute.

File: src/jsx/components/Modal/Radio.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+	return (
+		<div className="radio-wrapper">
+			<div className="radio-combo">
+				<input readOnly checked={!!checked} type="radio" id={value} value={value} name="age" />
+				<span className="radio-target" />
+			</div>
+			<label className="radio-label" htmlFor={value}>
+				{value}
+				{subValue ? <label className="radio-sub-value">{`(${subValue})`}</label> : null}
```
Comment (caionnx) [inline src/jsx/components/Modal/Radio.jsx]: Label should not be descendant of label. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label (under Technical summary)

Comment (miinow) [reply src/jsx/components/Modal/Radio.jsx]: It was careless, I have fixed it.

**Pull request merged**